5-Heroku/server/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const cors = require('cors')
+const cookieParser = require('cookie-parser')
+const todo_service = require('./todo_service')
+
+const app = express()
+const port = process.env.PORT || 3000;
+
+const items = {};
+
+app.use(cors({
```

Comment:
after you'll use webpack proxy you won't need this.
the rule of thumb is
`never write production code unless it has a use case in production`

in this case you don't need to allow cors for production, just for dev